File: CodeRed/src/app/model/address/address.ts
export class Address {
  public city: string;
  public country: string;
  public street: string;
  public zipCode: string;
  public attention: string;
  public careOf: string;
  public municipality: string;
  constructor(
    city: string,
    country: string,
    street: string,
    zipCode: string,
    attention: string,
    careOf: string,
    municipality: string
  ) {
    this.city = city;
    this.country = country;
    this.street = street;
    this.zipCode = zipCode;
    this.attention = attention;
    this.careOf = careOf;
    this.municipality = municipality;
  }
}

File: WarGamesAPI/Model/Address.cs
using System.ComponentModel.DataAnnotations;

namespace WarGamesAPI.Model;

public class Address
{
    [Key]
    public int UserId { get; set; }

    public string? City { get; set; }
    public string? Country { get; set; }
    public string? Street { get; set; }
    public string? ZipCode { get; set; }
    public string? Attention { get; set; }
    public string? Municipality { get; set; }
    public string? CareOf { get; set; }

    public User? User { get; set; } 
}


